common4j/src/main/com/microsoft/identity/common/java/exception/ClientException.java

```diff
@@ -82,6 +82,11 @@ public class ClientException extends BaseException {
      */
     public static final String SCOPE_EMPTY_OR_NULL = "scope_empty_or_null";
 
+    /**
+     * Emitted when the device Token is not present in successful response
```

Comment:
nit :P 
Emitted when the device Token is not present in successful response.

Reply:
done